lib/r10k/module/forge.rb

```diff
@@ -89,7 +90,7 @@ def status
     if not self.exist?
       # The module is not installed
       return :absent
-    elsif not @metadata.exist?
+    elsif not File.exist?(@path + 'metadata.json')
```

Comment:
Since `@path + 'metadata.json'` will create a Pathname object, we could do `(@path + 'metadata.json').exist?` or create the object and invoke exist on it. Stylistic change, do what you think is more readable.